### PR TITLE
Closes #322: Adding .figure-caption styles

### LINKED ISSUE #322: Update Fig Caption styles
Fig caption styles in bootstrap should be updated to match what we did in QS1.

Some CSS suggestions:

```
figcaption {
    border-bottom: 1px solid #eef1f1;
    padding-bottom: 0.9375em;
    padding-top: .5em;
}
```

## Conditions of satisfaction
- [ ] caption should have padding/spacing above and below
- [ ] bottom border should appear

### PR DESCRIPTION
**Pull request by smayden**
This PR adds custom padding and a border bottom to the .figure-caption class (matching QS1 styles)

Review site: https://review.digital.arizona.edu/arizona-bootstrap/issue/322/docs/2.0/content/figures/

### CODE REVIEW

**Review by mmunro-ltrr — APPROVED** [review]
Looked similar to the review site on a local build, & passed local linting checks: just wondering where the specific color came from (#eef1f1) since we don't seem to use it elsewhere.

File: scss/_custom-styles.scss [hunk 1]
```diff
@@ -254,3 +254,11 @@
   // stylelint-disable-next-line declaration-no-important
   border-width: .25rem !important;
 }
+
+// >> Figure caption.
+
+.figure-caption {
+  padding-top: .5em;
+  padding-bottom: .9375em;
+  border-bottom: 1px solid #eef1f1;
```
Comment (trackleft) [inline scss/_custom-styles.scss:263]: I think we should use a color var here?

**Review by trackleft — CHANGES_REQUESTED** [review]

File: scss/_custom-styles.scss (hunk 1 above)
Comment (trackleft) [inline scss/_custom-styles.scss:263]:
```suggestion
  border-bottom: 1px solid $gray-200;
```

Comment (trackleft) [inline scss/_custom-styles.scss:263]: Alternate color suggestion.
```suggestion
  border-bottom: 1px solid $hr-border-color;
```

**Comment by smayden** [comment]
Per suggestions by Chris and Martin, this had be altered to use an existing brand color already set in bootstrap (gray200).

Should be ready for final review

**Review by mmunro-ltrr — APPROVED** [review]
Re-checked locally: prefer the non-arbitrary color anyway.

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**